worker/interactions.go

```diff
@@ -1,16 +1,14 @@
 package worker
 
-func isSuccessfulInteraction(err error) bool {
-	// No error always means success.
-	if err == nil {
-		return true
-	}
-	// List of errors that are considered successful interactions.
+func shouldRecordPriceTable(err error) bool {
```

Comment:
The file is redundant after the rename, we should move this to `worker/pricetables.go`. We could also get rid of it entirely and have the `worker` implement the `HostStore` interface and have a `RecordPriceTables` of its own that skips it and launches the goroutine abstracting those details from the caller.